plugins/netskope/icon_netskope/actions/create_new_url_list/action.py

```diff
@@ -0,0 +1,21 @@
+import insightconnect_plugin_runtime
+
+from .schema import Component, CreateNewUrlListInput, CreateNewUrlListOutput, Input, Output
+
+
+class CreateNewUrlList(insightconnect_plugin_runtime.Action):
+    def __init__(self):
+        super(self.__class__, self).__init__(
+            name="create_new_url_list",
+            description=Component.DESCRIPTION,
+            input=CreateNewUrlListInput(),
+            output=CreateNewUrlListOutput(),
+        )
+
+    def run(self, params={}):
+        data = {
+            Input.NAME: params.get(Input.NAME),
+            "data": {Input.URLS: params.get(Input.URLS), Input.TYPE: params.get(Input.TYPE)},
+        }
+        response = self.connection.client.create_a_new_url_list(data)
+        return {Output.CREATED_URLLIST: response}
```

Comment:
Would you be able to return the value as 
`return {Output.CREATED_URLLIST: self.connection.client.create_a_new_url_list(data)}`
Or does it break the functionality when not storing the result as `response`?